pkg/gameservers/gameservercounter.go

```diff
@@ -29,11 +29,11 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
-// AllocationCounter counts how many Allocated and
+// GameServerCounter counts how many Allocated and
```

Comment:
nit: can we call it PerNodeCounter or something likethat? this is to avoid stuttering "gameservers.GameServerCounter", instead "gameservers.PerNodeCounter"

Reply:
I like it. That reads much better. Making this change!